server/client_manager.py

```diff
@@ -103,6 +103,14 @@ def __init__(self, server, transport, user_id, ipid):
                     self.server.config["wtce_floodguard"]["times_per_interval"]
                 )
             ]
+            self.ooc_counter = 0
+            self.ooc_mute_time = 0
+            self.ooc_time = [
+                x * self.server.config["ooc_floodguard"]["interval_length"]
```

Comment:
This will prevent the server from launching if server owners haven't updated their configuration yet.

Reply:
It will not prevent the server from launching, a default of 0 is set in client_manager.py in line 106